src/common/vars/is-snap.injectable.ts

```diff
@@ -3,12 +3,11 @@
  * Licensed under MIT License. See LICENSE in root directory for more information.
  */
 import { getInjectable } from "@ogre-tools/injectable";
-import packageJson from "../../../package.json";
 
-const packageJsonInjectable = getInjectable({
-  id: "package-json",
-  instantiate: () => packageJson,
+const isSnapInjectable = getInjectable({
```

Comment:
The name of this injectable could describe better what it does. The term "snap" is a tad generic.